components/logins/android/build.gradle

```diff
@@ -88,6 +88,9 @@ configurations {
 }
 
 dependencies {
+    // Part of the public API.
+    api project(':sync15-library')
```

Comment:
Hmm... Need to think if the issues the current sync15-library gradle set up (ugh) has are going to be a problem here. Since you've tested this in lockwise I'm guessing something is making it work regardless, so it's... probably fine?

Even if they are, an issue we could always backport the fixes from the megazord-only patch to avoid making this patch depend on that. Might want to do that anyway before adding more stuff that depends on sync15-library.

That or wait for the megazord-only patch to land, although it hasn't gotten review yet so.